lib/src/lib/components/form-fields/select-field/select-field.component.ts

```diff
@@ -34,29 +35,41 @@ export class SelectFieldComponent extends FormComponent<FormFieldSelect> impleme
 
   public constructor(translateService: TranslateService) {
     super(translateService);
-    this.addSubscription(
-      this.conditionalOptionsChange.pipe(switchMap(({ condition, value }) => this.getConditionalOptions(condition, value))),
```

Comment:
`this.addSubscription` is not needed anymore?

Reply:
It still is. Fixed :)